content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -6,24 +6,20 @@ weight: 20
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
 
 This page shows how to configure [Group Managed Service Accounts](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) (GMSA) for pods and containers that will run on Windows nodes. Group Managed Service Accounts are a specific type of Active Directory account that provides automatic password management, simplified service principal name (SPN) management, and the ability to delegate the management to other administrators across multiple servers.
 
-In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as custom resources. Windows pods, as well as individual containers within a pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.14, the only container runtime interface that supports GMSA for Windows workloads is Dockershim. Implementation of GMSA through CRI and other runtimes is planned for the future.
-
-{{< note >}}
-Currently this feature is in alpha state. While the overall goals and functionality will not change, the way in which the GMSA credspec references are specified in pod specs may change from annotations to API fields. Please take this into consideration when testing or adopting this feature.
-{{< /note >}}
+In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as custom resources. Windows pods, as well as individual containers within a pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.16, Dockershim and the Docker runtime supports GMSA for Windows workloads end-2-end. Support for GMSA through CRI has been implemented but implementation of GMSA support in other runtimes like ContainerD is under investigation.
```

Comment:
“end-2-end”? I think I'd stop that sentence at “workloads”.

Reply:
Although the investigation (mentioned in the last sentence) is current, I feel it still comes under [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).

I recommend rewording that last sentence, to (eg):
```markdown
You can use GMSA with runtimes that implement {{< glossary_tooltip term_id="cri" text="CRI" >}}.
```

Reply:
For now, I removed the CRI/containerd reference. Once that whole stack is functional (CRI + containerd/other runtimes), we can update things here. Right now I think this will just create confusion for end-users.